src/SMG/ManagerBundle/Controller/ManagersController.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+namespace SMG\ManagerBundle\Controller;
+
+use Symfony\Bundle\FrameworkBundle\Controller\Controller;
+use FOS\RestBundle\Controller\Annotations;
+use SMG\UserBundle\Entity\User;
+use FOS\RestBundle\View\View;
+use FOS\RestBundle\Controller\FOSRestController;
+use Symfony\Component\HttpFoundation\Response;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\ParamConverter;
+
+class ManagersController extends FOSRestController
+{
+    use Traits\TokenFromHeaderTrait;
+    use Traits\HandleUserTrait;
+
+    /**
+     * TODO: annotation actually required to make difference
+     * with the other API which has the same name
+     *
+     * @Annotations\Post("/admin/users")
```

Comment:
you don't the annotation once again :(

/admin should be the prefix of the bundle